### PR TITLE
provide mechanism to give authenticated access to a token

### PR DESCRIPTION
**Pull request by jesuspg**
#### REVIEWERS

@flopezag 
#### DESCRIPTION
- check that token is from user with role admin
#### TESTING

Locally tested

### CODE REVIEW

File: tests/unit/test_server_manager.py [hunk 1]
```diff
@@ -298,10 +299,10 @@ def test_get_status(self, m, glancesync):
         :param m: The request mock.
         :return: Nothing.
         """
-        m.get('http://cloud.lab.fiware.org:4730/v2.0/tokens/token', text=self.validate_info_v2)
-        m.post('http://cloud.lab.fiware.org:4730/v2.0/tokens', text=self.validate_info_v2)
+        m.get(KEYSTONE_URL + '/v2.0/tokens/token', text=self.validate_info_v2)
```
Comment (hmunfru) [inline tests/unit/test_server_manager.py]: esta bien con dos tokens/tokens??

Comment (jesuspg) [inline tests/unit/test_server_manager.py]: es un test, el segundo token es un string simulando un token

File: tests/unit/test_authorization_manager.py [hunk 2]
```diff
@@ -217,9 +257,115 @@ def test_check_token(self, m):
...
+        # Given
+        auth = AuthorizationManager(identity_url='http://fake_url', api_version=AUTH_API_V2)
+        roles_list = None
+        # When
+        result = auth._is_admin(roles_list)
+        # Then
+        self.assertFalse(result)
+
+    def test_raise_exception_with_user_not_admin_with_from_keystone_v3(self, m):
+        # Given
```
Comment (flopezag) [inline tests/unit/test_authorization_manager.py:335]: It seems that something is missing in this title:

test_raise_exception_with_user_not_admin_**with_from**_keystone_v3

**Comment by flopezag** [comment]
LGTM

**Pull request merged**